sql/2019/10_SEO/10_07a.sql

```diff
@@ -1,19 +1,14 @@
 #standardSQL
-
-# <title> and <meta description> present
-
+# 10_07a: <title> and <meta description> present
```

Comment:
I think the analysts will also want to know how many sites have both of these present